app/code/Transoft/Blog/Controller/Adminhtml/Post/NewAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Controller\Adminhtml\Post;
+
+use Magento\Backend\App\Action;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+
+/**
+ * Class NewAction
+ *
+ * New action controller class
+ */
+class NewAction extends Action implements HttpGetActionInterface
+{
+    const ADMIN_RESOURCE = 'Post';
```

Comment:
Should be **Transoft_Blog::blog_manage_items** or other resource

Reply:
fixed, now it works fine